src/federation_server.ts

```diff
@@ -239,9 +239,7 @@ export class FederationServer {
         } else {
           return Promise.reject(
             new BadResponseError(
```

Comment:
I think this is not related with the PR, but came because of prettier. @tyvdh do you have prettier locally?

Reply:
Like in my ide? I don’t think so.